src/scenes/Event/store.js

```diff
@@ -514,7 +503,7 @@ export default class EventStore {
     const { phase, resultSetterAddress, resultSetStartTime, isOpenResultSetting, consensusThreshold } = this.oracle;
     const { global: { syncBlockTime }, wallet } = this.app;
     const currBlockTime = moment.unix(syncBlockTime);
-    const currentWalletQtum = wallet.currentWalletAddress ? wallet.currentWalletAddress.qtum : 0;
+    const currentWalletQtum = wallet.currentWalletAddress ? wallet.currentWalletAddress.naka : 0;
```

Comment:
this logic is going to have to change. mainly where Qtum was being used such as Betting. We will now have to check NBOT balance vs NAKA balance in this case.

Reply:
currentWalletNaka